store/localstore/kv.go

```diff
@@ -121,14 +111,17 @@ func (s *dbStore) UUID() string {
 }
 
 func (s *dbStore) GetSnapshot(ver kv.Version) (kv.MvccSnapshot, error) {
-	s.snapLock.RLock()
-	defer s.snapLock.RUnlock()
+	s.mu.RLock()
+	closed := s.closed
+	s.mu.RUnlock()
```

Comment:
If that is true, we do not need the lock at all. Just use `if s.closed {...}`, behaviour is the same.